### PR TITLE
Support archived connectors in Docs

### PR DESCRIPTION
**Pull request by evantahler**
Screenshot 2024-02-20 at 10 03 32 AM

This PR adds support for Archived connectors which have been removed from the repo and relocated to https://github.com/airbytehq/connector-archive.  Learn more [here](https://docs.google.com/document/d/1OL6vVaje5S1ajhXeSaGW3WKnvLik6g0ZDXnaNVDtAJs/edit).

Re:
* https://github.com/airbytehq/airbyte/pull/35355
* https://github.com/airbytehq/airbyte/pull/35359
* https://github.com/airbytehq/airbyte/pull/35370

This PR:
* Explains archived connectors on `/integrations/connector-support-levels`
* Sorts archived connectors to the ends of any lists

### CODE REVIEW

File: docusaurus/src/components/HeaderDecoration.jsx [hunk 1]
```diff
@@ -51,13 +52,17 @@ export const HeaderDecoration = ({
...
+                {CHECK_ICON} airbyte_lib
+              </a>
+            )}
           </dd>
         </div>
         <div>
           <dt>Support Level</dt>
           <dd>
-            <a href="/project-overview/product-support-levels/">
+            <a href="/integrations/connector-support-levels/">
```
Comment (evantahler) [inline docusaurus/src/components/HeaderDecoration.jsx:65]: This page was moved

File: docusaurus/src/remark/docsHeaderDecoration.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
 const visit = require("unist-util-visit").visit;
-const { isPypiConnector } = require("../connector_registry");
```
Comment (evantahler) [inline docusaurus/src/remark/docsHeaderDecoration.js:2]: unused import

**Comment by nataliekwong** [comment]
Thanks for the tag! Approach looks reasonable to me.

**Comment by postamar** [comment]
LGTM, though I don't know how any of this stuff works.

**Review by postamar — APPROVED** [review]

**Pull request merged**